examples/examples.yaml

```diff
@@ -1,6 +1,7 @@
 - path: "models/file"
   slow: ["choropleth.py"]
   no_js: ["maps_cities.py", "maps.py", "trail.py"]
+  no_diff: ["graphs.py"]
```

Comment:
Looking at it, the only deterministic layout function is nx.circular_layout. Having only that layout reduces the example value too much in my opinion. Would you be opposed to an additional screenshot test of a circular layout example?